### PR TITLE
update my portfolio broken link

### PR DESCRIPTION
**Pull request by highflyer910**

### CODE REVIEW

File: sites/theamax.me.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
-title: 'theamax.me'
-url: 'https://theamax.me'
-tags: ['vr-photographer', 'front-end', 'developer']
+title: 'Thea's Portfolio
```
Comment (xdesro) [inline sites/theamax.me.md:2]:
```suggestion
title: 'Thea's Portfolio'
```

File: sites/theamax.me.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+url: 'https:highflyer910.github.io'
```
Comment (xdesro) [inline sites/theamax.me.md:3]:
```suggestion
url: 'https://highflyer910.github.io'
```

Comment (highflyer910) [reply sites/theamax.me.md]: Hey @xdesro , just fixed those silly mistakes:) Thank you!

**Comment by xdesro** [comment]
Love it, @highflyer910! Thanks again for the fresh PR!

**Pull request merged**